static_src/components/panel_row.jsx

```diff
@@ -15,9 +15,7 @@ const propTypes = {
   styleClass: React.PropTypes.oneOf(STYLES)
 };
 
-const defaultProps = {
-  styleClass: 'none'
```

Comment:
Since the propTypes specifies `styleClass` as an enum, if you don't pass a styleClass, won't you get an error?